src/LibraryManager/Providers/Unpkg/UnpkgCatalog.cs

```diff
@@ -178,45 +178,52 @@ private void GetFiles(JObject fileObject, List<string> files)
 
             (string name, string version) = LibraryIdToNameAndVersionConverter.Instance.GetLibraryNameAndVersion(libraryNameStart, _provider.Id);
 
+            int at = name.LastIndexOf('@');
```

Comment:
>name [](start = 21, length = 4)

Can you add a comment indicating what case you are handling with this? I assume only one @ is possible?

Reply:
Yes. This is to handle @ to intrigue version completion but not name completion. I will add a comment for it.

---
In reply to: [205549335](https://github.com/aspnet/LibraryManager/pull/312#discussion_r205549335) [](ancestors = 205549335)